mod/tagrm.php

```diff
@@ -20,34 +20,51 @@ function tagrm_post(App $a)
 		$a->internalRedirect($_SESSION['photo_return']);
 	}
 
-	$tag =  (x($_POST,'tag')  ? hex2bin(notags(trim($_POST['tag']))) : '');
-	$item_id = (x($_POST,'item') ? intval($_POST['item'])               : 0);
+	$tags = [];
+	foreach (defaults($_POST, 'tag', []) as $tag) {
+		array_push($tags, hex2bin(notags(trim($tag))));
```

Comment:
Why not use the more popular `$tags[] = hex2bin(notags(trim($tag)));` syntax?

Reply:
Didn't know about it before this PR.  But I'll change it.

Reply:
I figured. It's a correct, albeit a little antiquated way of pushing values into a list.